flink-cyber/metron-parser-chain/parser-chains-config-service/src/main/java/com/cloudera/parserchains/queryservice/model/describe/ConfigParamDescriptor.java

```diff
@@ -59,6 +59,11 @@ public class ConfigParamDescriptor {
    */
   private boolean required;
 
+  /**
+   * Defines whether the user is able to enter multiple values per parameter.
+   */
+  private boolean multipleValues;
```

Comment:
what is the difference between multipleVariables and multiple parameters?

Reply:
the 'multiple' param is for UI to change the way the value is handled. I didn't use it to not have multiple meanings per variable